tests/FSharp.Control.AsyncSeq.Tests/AsyncSeqTests.fs

```diff
@@ -205,6 +205,27 @@ let ``AsyncSeq.bufferByCount empty``() =
   let s' = s |> AsyncSeq.bufferByCount 2 |> AsyncSeq.toList 
   Assert.True(([] = s'))
 
+
+[<Test>]
+let ``AsyncSeq.bufferByTimeAndCount``() =      
+  let s = asyncSeq {
+    yield 1
+    yield 2
```

Comment:
Great work. Could you also please add a more systematic test that covers a wider range of input sizes and buffer sizes?  e.g. compare to the tests I've been adding recently which always have some kind of "for" loop at the outer to cover a range of scenarios.  thanks